open_humans/signals.py

```diff
@@ -225,3 +226,23 @@ def email_confirmed_cb(email_address, **kwargs):
     Send a user a welcome email once they've confirmed their email address.
     """
     send_welcome_email(email_address)
+
+
+def send_withdrawel_email(project, slug):
```

Comment:
typo, should be "withdrawal" throughout :)

I'm expecting to see a `@receiver` that's like `projectmember_pre_save`, that (a) sends an email to the project (maybe this should be (optionally) off if a webhook exists), and (b) POSTs data in a JSON format to the webhook/callback URL provided by the project (if provided) c.f. https://sendgrid.com/blog/whats-webhook/

Reply:
English is hard.